### PR TITLE
adds to 'Auditing and logging' section

### PR DESCRIPTION
**Pull request by fboxwala**
Also I added 'the' before Mathematics 3 because I thought it sounded better.

### CODE REVIEW

**Comment by fboxwala** [comment]
I didn't mean for this to be in the PR but now it is is that okay?

**Comment by ehashman** [comment]
@fboxwala: it's cool. If you want to separate it, just `checkout -b` to copy this branch and then go back to it and do a hard reset to `HEAD^` with a force push.

**Comment by fboxwala** [comment]
Eh I'll leave that in and remember for next time

File: content/pages/mua-summary.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+Title: MUA Summary
+Category: pages
+Status: hidden
+Slug: mua-summary
```
Comment (ehashman) [inline content/pages/mua-summary.md]: The title should slugify to this so you don't actually need to specify this. Also, pages isn't a category so you can get rid of that too :)

File: content/pages/mua-summary.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+Anyone with a user account to a WiCS machine must sign our [Machine Usage Agreement]({filename}/pages/mua.md),
+summarized below. Those with guest accounts do not need to sign the MUA, but they are still held
+to the rules and regulations outlined in it.
+
+# Acceptable and Unacceptable Use #
+
+The WiCS machines are to be used for learning, class work, personal projects, and 
+use in accordance with [WiCS aims]({filename}/pages/about.md), that does not violate
+any of WiCS’ usage policies.
```
Comment (ehashman) [inline content/pages/mua-summary.md]: Might not want to use an `&rsquo;` here, apostrophe should be fine.

File: content/pages/mua-summary.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+If you have a project of interest that does not fall into these categories, you may
+request special priority from wics-sys.
```
Comment (ehashman) [inline content/pages/mua-summary.md]: Priority? Do you mean permission?

File: content/pages/mua-summary.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+# User Responsibilities #
+
+All usage on WiCS machines is governed by our [CoC]({filename}/pages/code-of-conduct.md),
```
Comment (ehashman) [inline content/pages/mua-summary.md]: All usage _of_

File: content/pages/mua-summary.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+If you have a project of interest that does not fall into these categories, you may
+request special priority from wics-sys.
+
+# User Responsibilities #
+
+All usage on WiCS machines is governed by our [CoC]({filename}/pages/code-of-conduct.md), 
+our [MUA]({filename}/pages/mua.md), local laws, and University policy. Users are also
+responsible for being aware of and adhering to policy of machines connected to from 
+WiCS machines.
```
Comment (ehashman) [inline content/pages/mua-summary.md]: Maybe add that "This means, among other things, that machines may not be used for commercial purposes, including the hosting of corporate websites and mining cryptocurrency."

File: content/pages/mua-summary.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+WiCS is not responsible for any activity on user accounts. Users will be held accountable 
+for all damaging, unethical and illegal activity.
+
+Users should be considerate of others’ needs when using WiCS machines. Do not disrupt the 
+work of anyone else, and do not hog resources.
+
+# Security #
+
+Users are only allowed to access their own accounts. Do not attempt to gain access to another’s
```
Comment (ehashman) [inline content/pages/mua-summary.md]: s/allowed/permitted/ Also, another quote that could be an apostrophe

File: content/pages/mua-summary.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+account or personal file. 
+
+Do not attempt to obtain anyone else’s password, and do not give your password or access to your account to anyone else.
```
Comment (ehashman) [inline content/pages/mua-summary.md]: This line is a bit long, also quote
-> "provide access to your account"

File: content/pages/mua-summary.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+work of anyone else, and do not hog resources.
+
+# Security #
+
+Users are only allowed to access their own accounts. Do not attempt to gain access to another’s
+account or personal file. 
+
+Do not attempt to obtain anyone else’s password, and do not give your password or access to your account to anyone else. 
+
+Choose a password that you do not use anywhere else, and do not give an machine automatic access
```
Comment (ehashman) [inline content/pages/mua-summary.md]: "We recommend you choose"

.rhosts files are so old that's kind of a bad example, lol

File: content/pages/mua-summary.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+to your WiCS account (via .rhosts files etc.)
+
+In the event that a security problem is found, members of [wics-sys](mailto:wics-sys@lists.uwaterloo.ca)
```
Comment (ehashman) [inline content/pages/mua-summary.md]: "In the event that you discover a security problem," ... s/wics-sys/the Systems Committee/

File: content/pages/mua-summary.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+must be notified immediately. The problem should not be exploited, and it should not be made
+public until it is corrected.
+
+# Auditing and Logging #
+
+Wics-sys will keep logs on all machines for diagnostic and auditing purposes. On all machines
```
Comment (ehashman) [inline content/pages/mua-summary.md]: The Systems Committee will...

They are not all necessarily located under /var/logs and that's not relevant so I would recommend you remove that bit :)

File: content/pages/mua-summary.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+these are located under /var/logs.
+
+# Rights of the Systems Committee and the WiCS Executive #
+
+Wics-sys may examine any file or program that appears to be in violation of usage policies.
```
Comment (ehashman) [inline content/pages/mua-summary.md]: The Systems Committee

File: content/pages/mua-summary.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+These processes may be killed, stopped, or interrupted at the discretion of the Systems
```
Comment (ehashman) [inline content/pages/mua-summary.md]: <s>These</s> Processes

File: content/pages/mua-summary.md [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+Committee, and he owner of the process will be notified.
```
Comment (ehashman) [inline content/pages/mua-summary.md]: the owner

File: content/pages/mua-summary.md [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+
+The Systems Committee may revoke a user’s access to an account provided that written
```
Comment (ehashman) [inline content/pages/mua-summary.md]: apostrophe (if you're curious why I care about that at all, it's just a possible unicode compatibility thing so I'd like to stick to pure ASCII in these docs)

File: content/pages/mua-summary.md [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+
+# Rights of the Systems Committee and the WiCS Executive #
+
+Wics-sys may examine any file or program that appears to be in violation of usage policies.
+These processes may be killed, stopped, or interrupted at the discretion of the Systems
+Committee, and he owner of the process will be notified.
+
+The Systems Committee may revoke a user’s access to an account provided that written 
+explanation is given. Any violation of the MUA, CoC, University policy, or local law 
+may result in removal of access. Users may appeal to the WiCS committee for reinstatement
```
Comment (ehashman) [inline content/pages/mua-summary.md]: Previously, we've used WiCS committee to mean "big WiCS". I'd write out the full and proper form of which committee we're talking about here.

File: content/pages/mua-summary.md [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+of revoked access.
+
+The WiCS committee may update any policy with reasonable notice.
```
Comment (ehashman) [inline content/pages/mua-summary.md]: Make this reference to the committee consistent to the one above.

File: content/pages/mua-summary.md [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+
+# Summary of a Summary #
```
Comment (ehashman) [inline content/pages/mua-summary.md:66]: ++

File: content/pages/mua-summary.md [hunk 18]
```diff
@@ -0,0 +1,68 @@
...
+
+Use the WiCS machines responsibly, and be considerate of others. The WiCS Systems Committee
```
Comment (ehashman) [inline content/pages/mua-summary.md]: +Please

File: content/pages/mua.md [hunk 19]
```diff
@@ -84,7 +84,13 @@ it can be corrected.
...
 ## Auditing and Logging ##
 
-The WiCS Systems Committee keeps records ("logs") on our machines to
+The WiCS Systems Committee keeps records ("logs") on our machines to:
+
+1. Have diagnostic information to refer to when hardware and software issues arise
+2. Document machine activity
+3. Refer to for auditing purposes
+
+Logs can be found in the /var/logs directory on all machines
```
Comment (ehashman) [inline content/pages/mua.md]: This is true but not relevant, and also not all-inclusive. I'd remove that. I just told you that for flavour when you asked about logging :)

File: content/pages/mua.md [hunk 20]
```diff
@@ -84,7 +84,13 @@ it can be corrected.
 
 ## Auditing and Logging ##
 
-The WiCS Systems Committee keeps records ("logs") on our machines to
+The WiCS Systems Committee keeps records ("logs") on our machines to:
+
+1. Have diagnostic information to refer to when hardware and software issues arise
```
Comment (ehashman) [inline content/pages/mua.md]: Can we format this like the constitution? With semicolons at the end until the last line with a period!

Also I'd change this line to
"Collect and store diagnostic information to aid in debugging hardware and software issues;"

File: content/pages/mua.md [hunk 21]
```diff
@@ -84,7 +84,13 @@ it can be corrected.
...
+2. Document machine activity
```
Comment (ehashman) [inline content/pages/mua.md]: "Document machine usage and activity;"

**Comment by ehashman** [comment]
Hey Fatema, I gave your draft the review you requested. Please address feedback and I'll merge your next commit.

File: content/pages/mua-summary.md (hunk 2 above)
Comment (fboxwala) [inline content/pages/mua-summary.md]: What is the code for an apostrophe? I can't find it

neermind

File: content/pages/mua-summary.md (hunk 3 above)
Comment (fboxwala) [inline content/pages/mua-summary.md]: probably

File: content/pages/mua-summary.md [hunk 22]
```diff
@@ -0,0 +1,68 @@
+Title: MUA Summary
+Status: hidden
+
+Anyone with a user account to a WiCS machine must sign our [Machine Usage Agreement]({filename}/pages/mua.md),
```
Comment (ehashman) [inline content/pages/mua-summary.md]: s/to a/on a/

File: content/pages/mua-summary.md [hunk 23]
```diff
@@ -0,0 +1,68 @@
...
+summarized below. Those with guest accounts do not need to sign the MUA, but they are still held
```
Comment (ehashman) [inline content/pages/mua-summary.md]: Those 'using available guest accounts'

File: content/pages/mua-summary.md [hunk 24]
```diff
@@ -0,0 +1,68 @@
...
+to the rules and regulations outlined in it.
+
+# Acceptable and Unacceptable Use #
+
+The WiCS machines are to be used for learning, class work, personal projects, and 
+use in accordance with [WiCS aims]({filename}/pages/about.md), that does not violate
```
Comment (ehashman) [inline content/pages/mua-summary.md]: That does not violate any of WiCS' usage policies seems a bit redundant. Maybe

in accordance with WiCS' aims, the full MUA, university policy, and local law.

File: content/pages/mua-summary.md [hunk 25]
```diff
@@ -0,0 +1,68 @@
...
+any of WiCS' usage policies.
+
+If you have a project of interest that does not fall into these categories, you may
```
Comment (ehashman) [inline content/pages/mua-summary.md]: these categories, 'or you have questions as to whether the project is permitted under the various policies,'

File: content/pages/mua-summary.md [hunk 26]
```diff
@@ -0,0 +1,68 @@
...
+request special permission from wics-sys.
```
Comment (ehashman) [inline content/pages/mua-summary.md]: Can you s/wics-sys/the Systems Committee/g ?

File: content/pages/mua-summary.md [hunk 27]
```diff
@@ -0,0 +1,68 @@
...
+WiCS is not responsible for any activity on user accounts. Users will be held accountable 
+for all damaging, unethical and illegal activity.
+
+Users should be considerate of others' needs when using WiCS machines. Do not disrupt the 
+work of anyone else, and do not hog resources.
+
+# Security #
+
+Users are only permitted to access their own accounts. Do not attempt to gain access to another’s
+account or personal file.
```
Comment (ehashman) [inline content/pages/mua-summary.md]: personal file**s**

File: content/pages/mua-summary.md [hunk 28]
```diff
@@ -0,0 +1,68 @@
...
+
+# Security #
+
+Users are only permitted to access their own accounts. Do not attempt to gain access to another’s
+account or personal file. 
+
+Do not attempt to obtain anyone else's password, and do not give your password or provide 
+access to your account to anyone else. 
+
+We recommend you Choose a password that you do not use anywhere else, and do not give a
```
Comment (ehashman) [inline content/pages/mua-summary.md]: capital Choose lol

also a machine -> any machine

File: content/pages/mua-summary.md [hunk 29]
```diff
@@ -0,0 +1,68 @@
...
+We recommend you Choose a password that you do not use anywhere else, and do not give a 
+machine automatic access to your WiCS account.
+
+In the event that you discover a security problem, members of [The Systems Committee](mailto:wics-sys@lists.uwaterloo.ca)
+must be notified immediately. The problem should not be exploited, and it should not be made
+public until it is corrected.
+
+# Auditing and Logging #
+
+The Systems Committee will keep logs on all machines for diagnostic and auditing purposes.
```
Comment (ehashman) [inline content/pages/mua-summary.md:49]: These logs contain records of user and machine activity, and may be shared with University Computing or local law enforcement as required.

File: content/pages/mua-summary.md [hunk 30]
```diff
@@ -0,0 +1,68 @@
...
+
+# Rights of the Systems Committee and the WiCS Executive #
+
+The Systems Committee may examine any file or program that appears to be in violation of 
+usage policies. Processes may be killed, stopped, or interrupted at the discretion 
+of the Systems Committee, and he owner will be notified.
```
Comment (ehashman) [inline content/pages/mua-summary.md]: **t**he owner

**Comment by ehashman** [comment]
Sorry for all the new comments, there were a few new typos and I had a bit more feedback.

**Comment by fboxwala** [comment]
I think it's all better now, lemme know if I should squash the commits

**Comment by ehashman** [comment]
Yeah please do. If you want to do it in two commits (one for the typos and one with the summary file), that's good too.

**Comment by fboxwala** [comment]
squashed.

**Comment by ehashman** [comment]
Ahh I couldn't understand why the commit structure wasn't what I asked for until I realized I said the wrong thing. When I said "the typos" I actually meant the ones in the MUA file, not my requested corrections.

As is, could you pretty please squash these into one commit? I will understand if you say no and/or yell at me for this, lol >.<

**Comment by fboxwala** [comment]
blergh

**Comment by fboxwala** [comment]
squashed

**Pull request merged**